app/templates/application.hbs

```diff
@@ -1,7 +1,9 @@
 {{#es-header}}
   {{es-navbar links=links}}
 {{/es-header}}
-<main>
-{{outlet}}
+
+<main class="container">
```

Comment:
I'd like to try to avoid a class here if at all possible. but if we have to use one, how about something more specific, like .main-content or similar.

Also, can you add role="main" to this element.